data/2016/sampleData/defaultconfig.yaml

```diff
@@ -102,7 +103,6 @@ speedtree:
   maxTreeHits: 20 # maximum hits it takes to chop a tree
 
 construction:
-  allowPOIPlacement: false
```

Comment:
maybe just comment it and add in the comment that isNoBuildInPois exist now

Reply:
Done 👍